library/src/main/java/com/novoda/downloadmanager/lib/FileDownloadInfo.java

```diff
@@ -15,14 +12,12 @@
 import java.util.Collection;
 import java.util.Collections;
 import java.util.List;
-import java.util.concurrent.ExecutorService;
 
 /**
  * Stores information about an individual download.
  */
 class FileDownloadInfo {
 
-    public static final String EXTRA_EXTRA = "com.novoda.download.lib.KEY_INTENT_EXTRA";
```

Comment:
I think that might have been a bug we added ... this should be publicly visible For example:

https://github.com/novoda/mubi/blob/366a6ab4557cc04bdf8a4bf09fb7a8ba2f479814/android/src/main/java/com/mubi/spotlight/download/DownloadCompleteReceiver.java#L45